.github/workflows/javascript-tests.yml

```diff
@@ -12,7 +12,7 @@ on:
   pull_request:
 
 jobs:
-  build:
```

Comment:
Changed the name because it was confusing seeing this in the UI and on my terminal when tracking CI runs